src/component/ProductsFilter/ProductFilterFunctions.js

```diff
@@ -18,6 +32,13 @@ const getRatingFilter = (categoryFilterData, filterByRatingSlider) => {
   }
 };
 
+/***
+ * This function filters products based on price
+ * @params - array of products and "high to low" or "low to high" input selected by user
```

Comment:
Appreciation: Good that you're using jsdoc. It's makes it easy to figure out what the function in doing when someone sees this.

Reply:
thanks sudhip actually its inspired by your code only i saw it in one of your prs and implemented the same